src/SelfServe/Decorators.tsx

```diff
@@ -81,20 +139,23 @@ const addToMap = (...decorators: Decorator[]): PropertyDecorator => {
   };
 };
 
-export const OnChange = (
-  onChange: (
-    newValue: InputType,
-    currentState: Map<string, SmartUiInput>,
-    baselineValues: ReadonlyMap<string, SmartUiInput>
-  ) => Map<string, SmartUiInput>
-): PropertyDecorator => {
+/**
+ * Indicates the callback to be fired when the UI element corresponding to the property is changed.
+ */
+export const OnChange = (onChange: OnChangeCallback): PropertyDecorator => {
   return addToMap({ name: "onChange", value: onChange });
 };
 
+/**
+ * Indicates that the UI element corresponding to the property should have an Info bubble.
```

Comment:
>bubble [](http://example.com/codeflow?start=83&length=6)

If you were wondering what an info bubble is: it is the icon that looks like "i" which users click on to get more information about the UI element.